spanner/spanner_snippets/spanner/spanner_database_add_split_points.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_database_add_split_points]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+	adminpb "cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func addSplitpoints(w io.Writer, dbName string) error {
+	ctx := context.Background()
+
+	client, err := database.NewDatabaseAdminClient(ctx)
```

Comment:
nit: This would help differentiate between data client and admin client.
```suggestion
	adminClient, err := database.NewDatabaseAdminClient(ctx)
```